common.gypi

```diff
@@ -36,12 +37,20 @@
         'defines': [
           'NDEBUG'
         ],
+        'cflags': [
+         '-flto'
```

Comment:
Why? How will this help when libortools is not compiled in lto format?

Reply:
I guess it won't?  Like other changes, this is a default pulled in via `common.gypi`.  Assuming `-flto` does no harm if the linked libs aren't lto capable, then I'm inclined to just leave this here.